src/cgame/cg_gameinfo.cpp

```diff
@@ -41,11 +41,11 @@ CG_ParseInfos
 int CG_ParseInfos( const char *buf, int max, char *infos[] )
 {
 	char *token;
-	int  count;
+	int  count = 0;
 	char key[ MAX_TOKEN_CHARS ];
 	char info[ MAX_INFO_STRING ];
 
-	count = 0;
+	int infoPostfixLen = strlen( "\\num\\" ) + strlen( va( "%d", MAX_ARENAS ) );
```

Comment:
`infoPostfixLen` is used in:
`infos[ count ] = (char*) BG_Alloc( strlen( info ) + infoPostfixLen + 1 );`

BG_Allock take an `int` parameter, looks like it is better to change it
to std::size_t, since `malloc` / `memset` accept `size_t`, and using signed 
type looks like at least strange.

`void *BG_Alloc( int size )
{
	void *ptr = malloc( size );

	if ( ptr )
	{
		memset( ptr, 0, size );
	}

	return ptr;
}`

May be you have some thoughts about?